Vagrantfile

```diff
@@ -1,33 +1,70 @@
 # -*- mode: ruby -*-
 # vi: set ft=ruby :
 
+require 'yaml'
+
+ansible_vars = YAML.load_file('provisioning/vars.yml')
+
+app_name = ansible_vars["nodejs_app_name"]
+
+app_directory = ansible_vars["nodejs_app_install_dir"]
+
+# Check for the existence of 'VM_HOST_TCP_PORT' or 'VM_GUEST_TCP_PORT'
+# environment variables. Otherwise if 'nodejs_app_tcp_port' is defined
+# in vars.yml then use that port. Failing that use defaults provided
+# in this file.
+host_tcp_port = ENV["VM_HOST_TCP_PORT"] || ansible_vars["nodejs_app_tcp_port"] || 8080
```

Comment:
@jobara - the issue is one of usability. If the user "just changes nodejs_app_tcp_port in vars.yml" they should expect it to actually work - that is, the application should actually use that port. Whereas in fact what happens if they just change that entry, the application will fail to start up since it is in fact using a different port.
At the very least, there should be clear instructions on EVERYTHING the user needs to do to make a working instance of the app come up and actually run, inside the vagrant box on a different port - and, preferably, those instructions should consist of making only one change to a configuration file.